File: app_manager/appmanager/src/main/java/app/editors/manager/mvp/models/user/Module.java
package app.editors.manager.mvp.models.user;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Module {

    @SerializedName("webItemId")
    @Expose
    private String webItemId;

    @SerializedName("users")
    @Expose
    private List<User> users;

    @SerializedName("groups")
    @Expose
    private List<Group> groups;

    @SerializedName("enabled")
    @Expose
    private boolean enable;

    @SerializedName("isSubItem")
    @Expose
    private boolean subItem;

    public String getWebItemId() {
        return webItemId;
    }

    public void setWebItemId(String webItemId) {
        this.webItemId = webItemId;
    }

    public List<User> getUsers() {
        return users;
    }

    public void setUsers(List<User> users) {
        this.users = users;
    }

    public List<Group> getGroups() {
        return groups;
    }

    public void setGroups(List<Group> groups) {
        this.groups = groups;
    }

    public boolean isEnable() {
        return enable;
    }

    public void setEnable(boolean enable) {
        this.enable = enable;
    }

    public boolean isSubItem() {
        return subItem;
    }

    public void setSubItem(boolean subItem) {
        this.subItem = subItem;
    }
}

File: app_manager/core/src/main/java/app/documents/core/network/manager/models/user/Module.kt
package app.documents.core.network.manager.models.user

import app.documents.core.network.login.models.User
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Module(
    @SerializedName("webItemId")
    @Expose
    var webItemId: String? = null,

    @SerializedName("users")
    @Expose
    var users: List<User>? = null,

    @SerializedName("groups")
    @Expose
    var groups: List<Group>? = null,

    @SerializedName("enabled")
    @Expose
    var isEnable: Boolean = false,

    @SerializedName("isSubItem")
    @Expose
    var isSubItem: Boolean = false
)